README.md

```diff
@@ -70,7 +70,7 @@ $ npx vitest
 - [Unit Testing](./test/core)
 - [Vue Component Testing](./test/vue)
 - [React Component Testing](./test/react)
-- [React Mock Service Worker with Testing Library Testing](./test/msw-testing-lib-react)
+- [React Mock Service Worker with Testing Library Testing](./test/react-testing-lib-msw)
```

Comment:
`React Mock Service Worker with Testing Library Testing` is also a bit of a long one 😁  

Maybe rename to `React Component Testing with MSW` also to be more in line with the other titles?

Reply:
ye :fire: sounds good :100: 

I'm renaming it to **React Testing with MSW**, component would be consistent but we are doing a bit more than just testing a component :eyes: :sunglasses: :fire: